addons/mail/security/ir.model.access.csv

```diff
@@ -1,5 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_mail_message_all,mail.message.all,model_mail_message,,1,0,0,0
+access_mail_message_oembed,mail.message.oembed,model_mail_oembed,,1,0,0,0

```

Comment:
Anyway can read the embed from anyone? That should be linked to the visibility of the attachment

Reply:
All right. Doesn't change much for me.